scripts/setup_new_sdk.sh

```diff
@@ -7,7 +7,7 @@ CLIENTS_GENERATOR_DIR="${PWD}"
 CONFIG_PATH="${CLIENTS_GENERATOR_DIR}/config/clients/${SDK_ID:?}"
 SDK_OUTPUT_PATH="${CLIENTS_GENERATOR_DIR}/clients/${SDK_ID}"
 appLongName="OpenFGA"
-apiDocsUrl="https://openfga.dev/docs/api"
+apiDocsUrl="https://openfga.dev/api/service"
```

Comment:
```suggestion
apiDocsUrl="https://openfga.dev/api"
```

To be consistent with the other SDKs